config/default.json

```diff
@@ -12,7 +12,7 @@
     "service-2-service-auth": {
       "url": "http://localhost:4552",
       "totpSecret": "AAAAAAAAAAAAAAAA",
-      "microservice": "cmc_legal_frontend"
+      "microservice": "cmc"
```

Comment:
need to change the secret in `main.tf` to use the cmc one

Reply:
done